content/en/blog/2024/steering-committee-results-2024.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog
+title: "Announcing the 2024 Steering Committee Election Results"
+date: 2024-10-02
```

Comment:
A publication datetime is handy, but not required.

Reply:
Okay, I can edit one in.